apps/dashboard/redirects.js

```diff
@@ -284,6 +284,18 @@ function redirects() {
       destination: "/explore",
       permanent: false,
     },
+
+    // framer redirects
+    {
+      source: "/deploy",
+      destination: "/contracts/deployment-tool",
+      permanent: false,
+    },
+    {
+      source: "/smart-contracts",
+      destination: "/contracts/explore",
```

Comment:
jsut double checking this is the right destination?

Reply:
yes